src/core/tests/frontend/shared/test_builtin_extensions_1/builtin_extensions.cpp

```diff
@@ -7,7 +7,9 @@
 
 #ifdef ENABLE_OV_ONNX_FRONTEND
 #    include <openvino/frontend/onnx/extension/conversion.hpp>
-#    define ONNX_EXT std::make_shared<ov::frontend::onnx::ConversionExtension>("NewCustomOp_3", CustomTranslatorONNX),
+#    define ONNX_EXT                                                                                      \
+        std::make_shared<ov::frontend::onnx::ConversionExtension>("NewCustomOp_3", CustomTranslatorONNX), \
+            std::make_shared<ov::frontend::onnx::ConversionExtension>("Relu", ReluToSwishTranslatorONNX),
```

Comment:
Could we also check that `OPENVINO_FRAMEWORK_MAP` is supported?
https://github.com/openvinotoolkit/openvino/blob/master/docs/template_extension/new/identity.hpp#L24

Reply:
As I understand `OPENVINO_FRAMEWORK_MAP` feature on python level is realized via `OpExtension`. I have already tests for it in: https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_frontend/test_frontend_onnx.py#L467-L595
Do you have in mind some other scenario to test?